apm-agent-plugins/apm-java-concurrent-plugin/src/test/java/co/elastic/apm/agent/concurrent/AsyncTraceMethodInstrumentationTest.java

```diff
@@ -68,6 +66,7 @@ void setUp(TestInfo testInfo) {
         }
 
         tracer = mockInstrumentationSetup.getTracer();
+        assertThat(tracer.getConfig(CoreConfiguration.class).getTraceMethods()).containsExactly(testTraceMethodMatcher);
```

Comment:
[minor/style] I would just put this extra assertion just after the `doReturn` call so it's clear that it does not relates to the test scenario, but only the "test setup" phase.

Reply:
This is still in the setup phase.
I put it here because here I have the tracer, with which I can do `tracer.getConfig(CoreConfiguration.class).getTraceMethods()`, which is exactly what's going to be done next in `initInstrumentation`